src/components/chart/state-history-chart-timeline.ts

```diff
@@ -138,8 +138,12 @@ export class StateHistoryChartTimeline extends LitElement {
         "ui.components.history_charts.duration"
       )}: ${millisecondsToDuration(durationInMs)}`;
 
+      const markerLocalized = !computeRTL(this.hass)
+        ? marker
+        : '<span style="direction: rtl;display:inline-block;margin-right:4px;margin-inline-end:4px;border-radius:10px;width:10px;height:10px;background-color:#9e9e9e;"></span>';
```

Comment:
`background-color` is hard coded here. It should come from `params`.
I think it's better to wrap `marker` with an additional `span` and style that instead of replacing it entirely but I haven't tried it so not sure if it is feasible.

Reply:
@MindFreeze this was the original code which I think was copied from the source component. If you think this change is still necessary I'll do it. Let me know. Thanks

Reply:
@bramkragten  ? @MindFreeze  ?

Reply:
The color definitely has to be dynamic. If wrapping it with a span doesn't work, you should at least set `background-color` from `params`

Reply:
Fixed. Sorry I missed it and it took so long. Thanks. @MindFreeze @bramkragten